scripts/coder-dev.sh

```diff
@@ -26,4 +26,7 @@ if [[ ! -x "${CODER_DEV_BIN}" ]]; then
 	exit 1
 fi
 
+if [[ -x /tmp/coder ]]; then
```

Comment:
I'm not sure about sneaking this change?

Reply:
This wasn't supposed to be here, good catch!